ports/servoshell/egl/app_state.rs

```diff
@@ -569,6 +569,9 @@ impl RunningAppState {
             coordinates.viewport.width() as u32,
             coordinates.viewport.height() as u32,
         ));
+        let size = coordinates.viewport.size;
+        self.active_webview()
+            .move_resize(DeviceRect::from_size(size.to_f32()));
```

Comment:
```suggestion
        self.active_webview()            .move_resize(coordinates.viewport.size.to_f32().into());
```

Also we should move_resize before resize, to be consistent.